settings/Sparkpost.setting.php

```diff
@@ -17,6 +17,19 @@
     'description' => 'SparkPost REST API key',
     'help_text' => 'You can create API keys at: https://app.sparkpost.com/account/credentials',
   ),
+  'sparkpost_customCallbackUrl' => array(
+    'group_name' => CRM_Sparkpost::SPARKPOST_EXTENSION_SETTINGS,
+    'group' => 'com.cividesk.email.sparkpost',
+    'name' => 'sparkpost_customCallbackUrl',
+    'type' => 'String',
+    'html_type' => 'password',
```

Comment:
Why 'password' here? We do not need to blank out what is entered for this setting.

Reply:
oops, sorry, copy and paste error...

Reply:
Please fully test the PR before I merge it. This was an obvious mistake. Thanks.

Reply:
@nganivet. I have tested it and it works nicely. FYI that was not an obvious mistake - that html_type setting type value is not surfaced in the extension. the significant field, type was correctly set as String.